src/test/java/com/training/web/tests/rccl/GuestAccountTest.java

```diff
@@ -101,4 +106,48 @@ public void testCreateGuestAccountAndValidatePageElements() throws NullPointerEx
         softAssertions.assertThat(homePage.isPlanNewCruiseVisible()).as("Plan new cruises button not visible")
                 .isTrue();
     }
+
+    @Test
+    public void testAccountCreationGuiLoyaltyAddition() {
+
+        String lastName = "Poole";
+        String loyaltyId = "137529822";
+
+        CreateAccount createAccount = new LoginPage().createAccount();
+        createAccount.enterName(firstName, lastName)
+                .enterDateOfBirth()
+                .selectCountry()
+                .enterSecurityQuestion()
+                .enterIdPassword(email,password)
+                .acceptTerms();
+
+        homePage = createAccount.clickDone();
+
+        com.training.pojos.ga.authentication.Request request =
+                new com.training.pojos.ga.authentication.Request(email, password);
+
+        response = new RestEngine().getResponse(EMAIL_AUTHENTICATION_URL, headerMap,
+                new Gson().toJson(request));
+
+        com.training.pojos.ga.authentication.Response responseElement =
+                response.as(com.training.pojos.ga.authentication.Response.class);
+
+        com.training.pojos.ga.update.Request updateRequest = com.training.pojos.ga.update.Request.builder()
+                .vdsId(responseElement.getPayload().getAccountId())
+                .build();
+        headerMap.put(map.get("rcclLoyaltyHeaderName"), responseElement.getPayload().getAccessToken());
+
+        response = new RestEngine().getPutResponse("/v1/guestAccounts/loyalty", headerMap,
+                new Gson().toJson(updateRequest));
+
+        com.training.pojos.ga.update.Response updateResponse = response.as(com.training.pojos.ga.update.Response.class);
+
+        Assertions.assertEquals(200,Integer.parseInt(updateResponse.getStatus()));
+
+        getWebDriver().navigate().refresh();
```

Comment:
refresh is a method in selenide change the method name

Reply:
i am using selenide refresh